### PR TITLE
feat: add tracker

### PR DESCRIPTION
**Pull request by truxnell**

### CODE REVIEW

File: qbittools/commands/utils.py [hunk 1]
```diff
@@ -169,6 +169,12 @@
         'required_seed_ratio': 0,
         'required_seed_days': 3.5,
     },
+    'superbits': {
+        'name': 'superbits',
+        'urls': ['superbits.org','sptracker.cc','superbits.org'],
```
Comment (onedr0p) [inline qbittools/commands/utils.py]:
```suggestion
        'urls': ['sptracker.cc', 'superbits.org'],
```

Comment (onedr0p) [reply qbittools/commands/utils.py]: Looks like you had a dupe domain here

**Review by onedr0p — CHANGES_REQUESTED** [review]
See suggestion

**Comment by truxnell** [comment]
@onedr0p you're correct, fixed

**Review by onedr0p — APPROVED** [review]

**Pull request merged**